libs/util/billing-types/src/lib/billing-feature.types.ts

```diff
@@ -6,7 +6,7 @@ export enum BillingFeature {
   BranchPerResource = "feature-branch-per-resource",
   ChangeGitBaseBranch = "feature-change-git-base-branch",
   CodeGenerationBuilds = "feature-code-generation-builds",
-  CodeGeneratorName = "feature-code-generator-name",
+  CodeGeneratorDotNet = "feature-code-generator-dotnet",
```

Comment:
Writing this as a note to myself before merging this PR - we need to create the new entitlement on Stigg prod (and remove the old entitlement no longer in use)

Reply:
Done:
![image](https://github.com/amplication/amplication/assets/1912933/ee413772-ad2c-4495-99d8-8a40c859b2f6)